web/src/main/java/org/jboss/cdi/tck/shrinkwrap/ee/WebArchiveBuilder.java

```diff
@@ -94,7 +94,9 @@ protected WebArchive buildInternal() {
         processResources(webArchive);
 
         // Deployment descriptors
-        webArchive.addAsWebInfResource(getBeansDescriptorAsset(), buildBeansDescriptorTargetPath(getBeansDescriptorTarget()));
+        if (includeBeansXml) {
```

Comment:
Do we actually need to have two `WebArchiveBuilder` classes? There is one in `impl` and other under `web`; seems like a remnant of the TCK split.

Reply:
The one in in the web module has an ejb module that is included by default in test deployments and has a dependency on the EnterpriseArchiveBuilder.

Reply:
Perhaps so, I didn't really check.